.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "presets": ["env"],
```

Comment:
We've been using `babel-preset-es2015`; installing this library now encourages switching. See [here](https://babeljs.io/env).

I couldn't actually explain what this means.

Reply:
This only means that each year a new version of ES is released so instead of using the (outdated) year preset it should be a general `env` preset

Reply:
Thanks for the explanation